### PR TITLE
test tensor gradient and divergence

### PR DESCRIPTION
**Pull request by simonbyrne**
- [x] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

**Comment by szy21** [comment]
I pushed a draft for the test. Hope it makes sense. Please feel free to clean it up.

**Comment by simonbyrne** [comment]
bors try

File: test/Operators/finitedifference/tensor.jl [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    S = (∇ᵥuvw .+ adjoint.(∇ᵥuvw)) ./ 2
+    S_scalar = (∇ᵥuvw_scalar + adjoint(∇ᵥuvw_scalar)) / 2
+    S_ref = fill(S_scalar, fspace)
+    @test S ≈ S_ref
+
+    @test norm(S_scalar) ≈ norm(Geometry.components(S_scalar))
+    @test norm.(S) ≈ fill(norm(S_scalar), fspace)
+
+    # TODO: set boundary values
```
Comment (szy21) [inline test/Operators/finitedifference/tensor.jl:64]:
```suggestion

```

**Review by szy21 — APPROVED** [review]
Thanks Simon!

**Comment by szy21** [comment]
Should we squash the commits?
